README.MARCJOSEPH.md

```diff
@@ -0,0 +1,35 @@
+# Car Rules 
+general Usage Rules
+
+1.	The vehicle can only go to church, school, and restaurants (that are in a 20 mile radius from the home). All other destinations require permission.
+2. Passengers are limited to Mom and/or Dad. No other passengers (including siblings) are currently allowed. Exceptions require permission for each occurrence.
+3.	The car must be kept clean, no trash or food will be present upon exiting the vehicle.
+
+
+## Gas Maintenance and Fueling
+maintenance considerations
+
+1. Gas will be provided only for trips to and from school,church, work or job interviews; any personal outings are your financial responsibility. For personal outings not related to schooling or the above, fuel usage must be calculated in advance and paid for by you. Parents will contribute $100 for maintenance every two months and $60 for fuel weekly, the car may not be driven unless it meets maintenance standards. The car should never be brought home with less than 1/4 of a tank. Insurance for the car is your responsibility.
+### Scheduling 	
+
+
+
+1.	Your schedule must be edited and tuned for the day before using the car.
+2.  When leaving must leave 10 minutes prior then scheduled destinaation time e.g if maps says it will take 30 minutes to get there, then you will leave 40 minuites prior to your apponintment.
+## Revocation requirements 
+1.	Failure to follow the rules may result in revocation of vehicle usage privileges.
+2.	Vehicle usage privileges may be revoked based on a three-strike system:
+  - The first strike two strikes remain 
+  - The second strike one strike remains 
+  - Third strike after this complete revocation 
+    Any additional strikes will lead to a revocation up to 6 months of vehicle usage. Strikes will be issued if any rule is violated, and they reset every 6 months .
+## Ownership
+1.  Ultimate ownership of the vehicle is to the parents if parents need the vehicle priority is given to them over you. 
+2.	Any items in the car that belong to you, and are used, will remain your property. If the vehicle is revoked, any and all items inside the car that are yours must be returned or compensated for accordingly.
+
```

Comment:
lets make it clearer, that you have to remove your belongings and leave the car in a clean state.